scripts/rabbitmq-server.bat

```diff
@@ -112,6 +112,10 @@ if "!ENV_OK!"=="false" (
     EXIT /b 78
 )
 
+if "!RABBITMQ_SCHEDULING_BIND_TYPE_FLAG!"=="" (

```

Comment:
Not that I'm aware, but I set it here the same way `IO_THREAD_POOL_SIZE` is defined just [above](https://github.com/rabbitmq/rabbitmq-server/pull/873/files#diff-bf0f372f52d6c544a7ae5214db1c5f21L100)